DateLocal.java

```diff
@@ -0,0 +1,23 @@
+import java.time.*;
+import java.time.format.DateTimeFormatter;
+
+public class DateLocal {
+
+    static LocalDate myMethod(LocalDate date1, int days, int month, int year) {

```

Comment:
```suggestion
    static LocalDate myMethod(LocalDate currentDate, int days, int month, int year) {
```
btw date1 value never used in method

Reply:
fixed on 2c2bb34